docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,297 @@
+
+The Domain of the CR is `theirs.com`, the Group is `theirgroup` and the kind and go type would be `ExternalType`.
+
+If there is an interest to have multiple Controllers running in different Groups (e.g. because one is an owned CRD and one is an external Type), please first
+reconfigure the Project to use a multi-group layout as described in the [Multi-Group documentation](../migration/multi-group.md).
+
+### Prerequisites
+
+The following guide assumes that you have already created a project using `kubebuilder init` in a directory in the GOPATH. Please reference the [Getting Started Guide](../getting-started.md) for more information.
+
+Note that if you did not pass `--domain` to `kubebuilder init` you will need to modify it for the individual api types as the default is `my.domain`, not `theirs.com`.
+Similarly, if you intend to use your own domain, please configure your own domain with `kubebuilder init` and do not use `theirs.com for the domain.
+
+### Add a controller for the external Type
+
+We will now add a controller for the given type
+
+```shell
+kubebuilder create api --group theirgroup --version v1alpha1 --kind ExternalType --controller --resource=false
+```
+
+Note that the `resource` argument is set to false, as we are not attempting to create our own CustomResourceDefinition,
+but instead rely on an external one.
+
+### Adjusting the domain for an external type
+
+In case of a domain mismatch of your project domain with the domain of the external type, you will need to adjust the domain for the external type:
+
+#### Using a type not included in the Kubernetes Core Resources
+
+file: PROJECT
+```
+domain: my.domain
+layout:
+- go.kubebuilder.io/v4
+projectName: testkube
+repo: github.com/jakobmoellerdev/testkube
+resources:
+- controller: true
+  domain: theirs.com
+  group: mygroup
+  kind: ExternalType
+  version: v1alpha1
+version: "3"
+```
+
+Adjusting the generated RBAC manifests:
+
+file: internal/controller/externaltype_controller.go
+```go
+// ExternalTypeReconciler reconciles a ExternalType object
+type ExternalTypeReconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+//+kubebuilder:rbac:groups=theirgroup.theirs.com,resources=externaltypes,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=theirgroup.theirs.com,resources=externaltypes/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=theirgroup.theirs.com,resources=externaltypes/finalizers,verbs=update
+
+```
+
+#### Using a type included in the Kubernetes Core Resources
+
+file: PROJECT
+```
+domain: my.domain
+layout:
+- go.kubebuilder.io/v4
+projectName: testkube
+repo: github.com/jakobmoellerdev/testkube
+resources:
+- controller: true
+  domain: ""                                                                                                                                                                                             
+  kind: Pod
+  version: v1alpha1
+version: "3"
```

Comment:
It is no longer required for CoreTypes, see; 

https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4-multigroup/PROJECT#L86-L90